rskj-core/src/main/java/co/rsk/peg/federation/FederationStorageProvider.java

```diff
@@ -40,4 +41,7 @@ public interface FederationStorageProvider {
     void setLastRetiredFederationP2SHScript(Script lastRetiredFederationP2SHScript);
 
     void save(NetworkParameters networkParameters, ActivationConfig.ForBlock activations);
+
+    void setFundTransactionUnsignedHash(Sha256Hash hash);
+
```

Comment:
move it to above the `save` method please :)

Reply:
Updated.